exonum-light-client/src/main/java/com/exonum/client/ExonumClient.java

```diff
@@ -40,6 +40,29 @@
    */
   HashCode submitTransaction(TransactionMessage tx);
 
+  /**
+   * Returns a number of unconfirmed transactions those are currently located in
+   * the memory pool and are waiting for acceptance to a block.
+   * @throws RuntimeException if the client is unable to complete a request
+   *        (e.g., in case of connectivity problems)
+   */
+  int getUnconfirmedTransactions();
+
+  /**
+   * Returns <b>true</b> if the node is connected to the other peers.
+   * And <b>false</b> otherwise.
```

Comment:
is it required to write at each method that it works in a blocking way? 🤔

Reply:
I'd put that in a class level comment (unless we make a hybrid with both blocking and non-blocking APIs)